kayenta-web/src/main/java/com/netflix/kayenta/filters/KayentaCorsFilter.java

```diff
@@ -0,0 +1,31 @@
+package com.netflix.kayenta.filters;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class KayentaCorsFilter implements Filter {
+
+  public KayentaCorsFilter() {}
+
+  @Override
+  public void init(FilterConfig filterConfig) throws ServletException {}
+
+  @Override
+  public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain) throws IOException, ServletException {
+    HttpServletResponse response = (HttpServletResponse) res;
+    HttpServletRequest request = (HttpServletRequest) req;
+    String origin = request.getHeader("Origin");
+    if (origin != null) {
```

Comment:
Could `origin` be an empty string?

Reply:
Hmm, I will add a check for that as well, and push it.  I'm not sure what would happen there, but 
I suspect it would "just work" as well as anything else.